go/vt/schema/ddl_strategy_test.go

```diff
@@ -123,6 +124,13 @@ func TestParseDDLStrategy(t *testing.T) {
 			runtimeOptions:       "",
 			isPostponeCompletion: true,
 		},
+		{
+			strategyVariable:    "online --in-order-completion",
+			strategy:            DDLStrategyOnline,
```

Comment:
Don't we also want this for the vitess strategy?

Reply:
`vitess` == `online`. They are synonyms with the intention of only using `vitess`. But changing names is hard.